ios/RNZoomUs.m

```diff
@@ -210,6 +236,10 @@ - (void)onMeetingReturn:(MobileRTCMeetError)errorCode internalError:(NSInteger)i
 - (void)onMeetingStateChange:(MobileRTCMeetingState)state {
   NSLog(@"onMeetingStatusChanged, meetingState=%d", state);
 
+  if (state == MobileRTCMeetingState_InMeeting) {
+    [self connectAudio];
```

Comment:
auto-connect should optional and possible to switch on/off in `ZoomUs.initialize(`